sites/docs/pages/components/custom-components.md

```diff
@@ -27,7 +27,7 @@ We'd love to see what you've built, and may add generally applicable components
 
 If you were creating a component called `<Hello />`, which included some text and a BarChart, to use in `index.md`, you could do so like this:
 
-Add a folder called `components/` in the root of your project. This is where Evidence looks for your Svelte components:
+Add a folder called `components/` in the root of your app directory. This is where Evidence looks for your Svelte components:
```

Comment:
This is the one scenario where I am unsure about the change. When you are referring to the directory of files. Is this still an app, or is this a _project_ that you use to build your app...

Reply:
perhaps `app workspace`